sysboundary/ionosphere.cpp

```diff
@@ -1207,13 +1361,33 @@ namespace SBC {
       TracingFieldFunction& BFieldFunction,
       bool outwards
    ) {
+      bool reTrace;
+      uint32_t attempts=0;
       switch(method) {
          case Euler:
             eulerStep(x, v,stepsize, BFieldFunction, outwards);
             break;
+         case ADPT_Euler:
+            do{
+               reTrace=!adaptiveEulerStep(x, v, stepsize, maxStepsize, BFieldFunction, outwards);
+               attempts+=1;
+            }while(reTrace && attempts<= Ionosphere::max_dormand_prince_attempts);
+            if (reTrace){
+               std::cerr<<"Adaptive Euler field line tracer exhausted all available attempts and still did not converge..."<<std::endl;
```

Comment:
What would we want the code to do in this situation?
Abort? (Because otherwise we have a physically nonsensical mapping)
Just continue?
Keep comlaining every timestep? (I guess that's what it would do right now)

Reply:
Yeah I am not sure and maybe we need to think about it. I would lean towards aborting because we would not want to proceed in the simulation with bad mapping. What do you say?